CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 <!-- For unreleased changes, see entries in .chloggen -->
 <!-- next version -->
 
+## [0.99.1] - 2024-05-09
+
+### 💡 Enhancements 💡
+
+- `agent`: Add an option `skipInitContainers` to skip init container setting file ACLs when the `runAsUser` and
+  `runAsGroup` are provided. This is useful when the user wants to manage the file ACLs themselves.
+  ([#1286](https://github.com/signalfx/splunk-otel-collector-chart/pull/1286))
+
```

Comment:
This change would also be part of this release. We can add this now or later, either way I'll do any cleanup afterwards if needed.
```suggestion
- `operator`: Bump dotnet to v1.5.0 in helm-charts/splunk-otel-collector/values.yaml ([#1282](https://github.com/signalfx/splunk-otel-collector-chart/pull/1282))

```